### DISCUSSION TITLE
Solving a 2d Fokker Planck equation

### DISCUSSION THREAD

**Discussion by naoufalEB** [zwicker-group/py-pde#567] [Q&A]
Dear Dr. Zwicker,
Thanks for this wonderful package! I really appreciate your work.
I tried adapting the answer you provided for a 1d Fokker-Planck [here](https://github.com/zwicker-group/py-pde/discussions/166) to a 2d Fokker Planck Equation like this one : 
<img width="185" alt="input_gemini" src="https://github.com/zwicker-group/py-pde/assets/95585501/a5f000bf-f71d-423c-91b6-ab4e5caf683a">
but was not successful. Could you please help

**Comment by david-zwicker** [comment]
I unfortunately don't have the bandwidth to solve your problem in detail, but I'm happy to help if you run into problems of using the package, which are not easily solvable from the information given in the documentation.
